src/capture/recognition/es/classifier_features.cc

```diff
@@ -0,0 +1,3130 @@
+#include "capture/recognition/es/classifier_features.h"
+
+#include <cstddef>
+#include <cstdint>
+
+const std::size_t RED_MEAN_SIZE = 768;
```

Comment:
A kind of bad points in copying is to duplicate those constant variables.
It can waste huge amount of memory.